src/remodal.js

```diff
@@ -502,15 +502,23 @@
       namespacify('is-initialized') + ' ' +
       remodal.settings.modifier + ' ' +
       namespacify('is', STATES.CLOSED));
-
-    remodal.$wrapper = $('<div>')
-      .addClass(
-        namespacify('wrapper') + ' ' +
+    var $existingWrapper = remodal.$modal.parent('.' + namespacify('wrapper'));
```

Comment:
Move `var $existingWrapper;` to the top of the function after all declarations, please, and an empty lines before and after, please, please, please. :cake: